pkg/generators/forwarding/fluentd/templates.go

```diff
@@ -596,3 +597,26 @@ const storeSyslogTemplate = `{{- define "storeSyslog" }}
 	severity debug
 </store>
 {{- end}}`
+
+const storeKafkaTemplate = `{{- define "storeKafka" }}
+    @type kafka2
+    brokers {{.Brokers}}
+    topic_key {{.Topic}}
+    default_topic {{.Topic}}
+    <format>
+       @type json
+    </format>
+    <buffer {{.Topic}}>
+        @type file
+        path '{{.BufferPath}}'
+        flush_interval "#{ENV['ES_FLUSH_INTERVAL'] || '1s'}"
```

Comment:
Remove ref to env vars

Reply:
All env vars or `ES_FLUSH_INTERVAL` only?

Reply:
All `ES_` prefixed env vars are removed. I kept the `BUFFER_` prefixed env vars which are set in run.sh